t/op/index.t

```diff
@@ -131,13 +131,15 @@ sub run_tests {
 
     SKIP: {
         skip("Not a 64-bit machine", 3) if length sprintf("%x", ~0) <= 8;
+        no warnings qw(non_unicode portable);
         my $a = eval q{"\x{80000000}"};
         my $s = $a.'defxyz';
         is(index($s, 'def'), 1, "0x80000000 is a single character");
 
         my $b = eval q{"\x{fffffffd}"};
         my $t = $b.'pqrxyz';
         is(index($t, 'pqr'), 1, "0xfffffffd is a single character");
+        use warnings;
```

Comment:
IMO using a code block when reducing warnings makes it more clear and avoid the need to reenable the warnings later, as they are default. But this is a style preference and not a blocker for such changes

Reply:
> IMO using a code block when reducing warnings makes it more clear and avoid the need to reenable the warnings later, as they are default. But this is a style preference and not a blocker for such changes

I think you'll see that I generally do place warnings modifications within a block.  But sometimes this stuff gets soooo tedious.

Done.